### ISSUE TITLE
Dashboard layout tutorial: Can't use 'anchor' as it is omitted from config type.

### ISSUE THREAD

**Issue by LukaszGrela** [siriwatknp/mui-treasury#1056]
I followed the [dashboard layout tutorial](https://mui-treasury.com/layout/tutorials/dashboard-layout/) and following code fails as the `config` param is typed as `(breakpoint: Breakpoint, config: Omit<TemporarySidebarConfig, 'id' | 'anchor' | 'variant' | 'hidden'>) => IEdgeSidebarRegistry;` where the `anchor` is omitted
```
scheme.configureEdgeSidebar((builder) => {
  builder
    .create("unique_id", { anchor: "left" })
    .registerTemporaryConfig("xs", {
      anchor: "left",
      width: "auto", // 'auto' is only valid for temporary variant
    })
    .registerPermanentConfig("md", {
      width: 256, // px, (%, rem, em is compatible)
      collapsible: true,
      collapsedWidth: 64,
    });
});
```

the error:

```
Argument of type '{ anchor: string; width: string; }' is not assignable to parameter of type 'Omit<TemporarySidebarConfig, "id" | "hidden" | "anchor" | "variant">'.
  Object literal may only specify known properties, and 'anchor' does not exist in type 'Omit<TemporarySidebarConfig, "id" | "hidden" | "anchor" | "variant">'.
```

I have removed offending line and it seems to work.

**TypeScript:** 4.2.3
**@mui-treasury/layout:** 4.5.0

**Comment by siriwatknp** [comment]
Thanks, seems like the docs need to be fixed (remove `anchor: "left"` from `registerTemporaryConfig`). Do you want to send a PR?

**Comment by hemmingson** [comment]
Hi team, trying to fix this issue and send a [PR](https://github.com/siriwatknp/mui-treasury/pull/1157).

**Comment by ShubhankarKG** [comment]
Hi team, can we close this issue? I stumbled upon this from Github's Explore option, but this has already been fixed in #1157
